Apps/Examples/Examples/All Examples/Annotations/CircleAnnotationExample.swift

```diff
@@ -32,9 +31,3 @@ final class CircleAnnotationExample: UIViewController, ExampleProtocol {
         finish()
     }
 }
-
```

Comment:
I think these are still useful in demonstrating a way to receive events when annotations are tapped.